src/ReactionService.Mappers/Db/DbReactionMapper.cs

```diff
@@ -0,0 +1,38 @@
+﻿using LT.DigitalOffice.Kernel.Extensions;
+using LT.DigitalOffice.ReactionService.Mappers.Db.Interfaces;
+using LT.DigitalOffice.ReactionService.Models.Db;
+using LT.DigitalOffice.ReactionService.Models.Dto.Requests;
+using Microsoft.AspNetCore.Http;
+using System;
+
+namespace LT.DigitalOffice.ReactionService.Mappers.Db;
+
+public class DbReactionMapper : IDbReactionMapper
+{
+  private readonly IHttpContextAccessor _httpContextAccessor;
+
+  public DbReactionMapper(
+    IHttpContextAccessor httpContextAccessor)
+  {
+    _httpContextAccessor = httpContextAccessor;
+  }
+
+  public DbReaction Map(CreateReactionRequest request, Guid imageId)
+  {
+    Guid reactionId = Guid.NewGuid();
```

Comment:
you use it once, so don't need to create variable
line 22 - delete
line 28: Id=Guid.NewGuid()

Reply:
Done